Sources/TestingMacros/Support/Additions/FunctionDeclSyntaxAdditions.swift

```diff
@@ -142,11 +142,12 @@ extension FunctionParameterSyntax {
     // Enclose the base type declaration reference in a 1-element tuple, e.g.
     // `(<baseType>)`. It will be used in a member access expression below, and
     // some types (such as function types) require this.
-    let metatypeMemberAccessBase = TupleExprSyntax {
-      LabeledExprListSyntax {
-        LabeledExprSyntax(expression: baseTypeDeclReferenceExpr)
-      }
-    }
+    //
+    // Intentionally avoid using the result builder variant of these APIs due to
```

Comment:
Is there a swift-syntax or swift PR/issue you can point to here? For future reference.

Reply:
Nobody involved in the investigation was sure what exactly the problem was, but in any event, the bug itself has long been fixed. We believe it was fixed in Swift 5.10. The problem is Swift's CI, in this case at least, is building some early dependencies using Swift 5.9 or earlier (intentionally), and that compiler had the bug.

Reply:
Alrighty. And the bug _only_ affected _this_ use of a result builder? How strange!